oxidation/libparsec/crates/crypto/Cargo.toml

```diff
@@ -7,34 +7,33 @@ license = " BUSL-1.1"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["use-rustcrypto"]
 use-sodiumoxide = [
-    "sodiumoxide",
-    "libsodium-sys",
-    "openssl",
-    "zeroize",
+    "dep:sodiumoxide",
+    "dep:libsodium-sys",
+    "dep:openssl",
+    "dep:zeroize",
 ]
 use-rustcrypto = [
```

Comment:
> can we remove the listing of the feature use-rustcrypto ?

That's an interesting point, at first I felt keeping `use-rustcrypto` make thing more explicit, but now you're pointing this out this may be misleading: the user may think not passing`use-rustcrypto` and passing it `use-rustcrypto` have different behavior (for instance automatically choosing the right between `use-rustcrypto` and `use-sodiumoxide` depending on compilation profile / platform).

I'm changing it to only have `use-sodiumoxide`.

done.